### PR TITLE
Fix Jingle Bells and added dependencies for it

### PR DESCRIPTION
**Pull request by sk82jack**
Fixed the octaves, note lengths and tempo for jingle bells but it required adding in a rest note and fixing the multiplier variable to work with pipelines

### CODE REVIEW

File: Resolve-Note.ps1 [hunk 1]
```diff
@@ -4,7 +4,7 @@ function Resolve-Note {
     param(
         [Parameter(Position = 0, Mandatory, ValueFromPipeline,ValueFromPipelineByPropertyName)]
         [Alias('♪', '♫')]
-        [ValidateSet('C', 'C#', 'Db', 'D', 'D#', 'Eb', 'E', 'F', 'F#', 'Gb', 'G', 'G#', 'Ab', 'A', 'A#', 'Bb', 'B')]
+        [ValidateSet('C', 'C#', 'Db', 'D', 'D#', 'Eb', 'E', 'F', 'F#', 'Gb', 'G', 'G#', 'Ab', 'A', 'A#', 'Bb', 'B', '-')]
```
Comment (vexx32) [inline Resolve-Note.ps1]: I'd recommend using `R` for a rest note.

**Review by vexx32 — COMMENTED** [review]
`-` is a bit easy to miss, especially in a crowd of notes :/

**Review by steviecoaster — CHANGES_REQUESTED** [review]
Please 'R' for the rest not, as '-' is very ambiguous, otherwise this looks great!

**Review by steviecoaster — APPROVED** [review]
Much more gooder

**Pull request merged**